data/locale/en-US.ini

```diff
@@ -7,7 +7,7 @@ NDIPlugin.NDIFrameSync="Framesync (experimental)"
 NDIPlugin.SourceProps.HWAccel="Allow hardware acceleration"
 NDIPlugin.SourceProps.AlphaBlendingFix="Fix alpha blending (adds a filter to this source)"
 NDIPlugin.SourceProps.ColorRange="YUV Range"
-NDIPlugin.SourceProps.ColorRange.Partial="Partial"
+NDIPlugin.SourceProps.ColorRange.Partial="Limited"
```

Comment:
This is literally "Partial" in OBS code:
* https://github.com/obsproject/obs-studio/blob/master/libobs/media-io/video-io.h#L117

Why change from "Partial" to "Limited"?

Reply:
Yes, in the code it is still "Partial", but all the obs-studio locale files use "Limited".
See [this search](https://github.com/search?q=repo%3Aobsproject%2Fobs-studio+path%3A**%2Fen-US.ini+language%3AINI+limited&type=code&ref=advsearch) vs. [this search](https://github.com/search?q=repo%3Aobsproject%2Fobs-studio+path%3A**%2Fen-US.ini+language%3AINI+partial&type=code&ref=advsearch).

The naming was changed in https://github.com/obsproject/obs-studio/pull/7215.